code/game/objects/items/stacks/rods.dm

```diff
@@ -9,7 +9,7 @@
 	throwforce = 15.0
 	throw_speed = 5
 	throw_range = 20
-	matter = list(DEFAULT_WALL_MATERIAL = 1875)
+	matter = list(DEFAULT_WALL_MATERIAL = SHEET_MATERIAL_AMOUNT / 2)

```

Comment:
BYOND doesn't have int at all IIRC, all is float. That's why you can see 1.0 values in old vars, they come from decompiled code.